packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.css

```diff
@@ -0,0 +1,49 @@
+.saltTrackerStep {
+  --trackerStep-icon-active-color: var(--saltTrackerStep-icon-active-color, var(--salt-status-info-foreground));
+  --trackerStep-icon-completed-color: var(--saltTrackerStep-icon-completed-color, var(--salt-status-success-foreground));
```

Comment:
```suggestion
  --trackerStep-icon-color-active: var(--saltTrackerStep-icon-color-active, var(--salt-status-info-foreground));
  --trackerStep-icon-color-completed: var(--saltTrackerStep-icon-color-completed, var(--salt-status-success-foreground));
```

Reply:
Is this something we're consistent on? Think I was following the naming of the theme tokens e.g. `status-success-foreground` vs `status-foreground-success`